csfieldguide/static/css/style.css

```diff
@@ -0,0 +1,15 @@
+img.chapter-icon {
+  fill: red;
```

Comment:
I can't see where this is used?

![localhost_8081_en-gb_chapters_](https://user-images.githubusercontent.com/8001048/34169711-1fe14e44-e54d-11e7-9d6a-c2536d9d3b4d.png)

Reply:
oh sorry, keep accidentally leaving in my tests